javascript/javascript1/week3/app.js

```diff
@@ -0,0 +1,178 @@
+// Item array removal
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+names.splice(1, 1);
```

Comment:
You should write a function that would be able to take in any name and remove it from list, right now you are just removing any item